includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -775,6 +775,8 @@ private function get_failure_message_from_error( $error ): string {
 				return __( 'Microdeposit transfers failed. Please check the account, institution and transit numbers.', 'woocommerce-payments' );
 			case 'payment_method_microdeposit_verification_attempts_exceeded':
 				return __( 'You have exceeded the number of allowed verification attempts.', 'woocommerce-payments' );
+			case 'payment_intent_mandate_invalid':
```

Comment:
Can we save these error codes as some constants above?

Reply:
We can. This is a great idea. Since this PR is already merged, I can address that in another PR or open an issue for that.